FMcommands/delete.py

```diff
@@ -25,16 +39,29 @@ def run(self, paths=None, *args, **kwargs):
 
         self.paths = paths or [self.view.file_name()]
         if get_settings().get("ask_for_confirmation_on_delete") is not False:
+            confirm_text = "Send {0}item{1} to trash".format(
+                ("{0} ".format(
+                    len(self.paths)) if len(self.paths) > 1 else ""),
+                ("s" if len(
+                    self.paths) > 1 else "")
+            )
+            cancel_text = "Cancel deletion of {0}file{1}".format(
+                ("{0} ".format(
+                    len(self.paths)) if len(self.paths) > 1 else ""),
+                ("s" if len(
+                    self.paths) > 1 else "")
+            )
+            paths_to_display = [
+                ["Confirm", confirm_text],
+                ["Cancel", cancel_text]
+            ]
+
+            for path in paths:
+                paths_to_display.append(
+                    [path.split(os.path.sep)[-1], path])
```

Comment:
Honestly, that's fine. But it reads a bit better if you use `os.path.basename` instead of `path.split(os.path.sep)[-1]`.

Reply:
Didn't even realize that was a thing! Done 👍